docs/Deploying-Pega-on-EKS.md

```diff
@@ -434,6 +434,7 @@ To configure the use of an Amazon AWS ALB ingress controller in the addons.yaml
 - Specify your EKS cluster name in the `clusterName: <YOUR_EKS_CLUSTER_NAME>` parameter.
 - Specify the region of your EKS cluster name in the `region: <YOUR_EKS_CLUSTER_REGION>` parameter. Resources created by the ALB Ingress controller will be prefixed with this string.
 - Specify the the AWS VPC ID of your EKS cluster name in the `VpcID: <YOUR_EKS_CLUSTER_VPC_ID>` parameter. You must enter your VPC ID here if ec2metadata is unavailable from the controller pod.
+- Uncomment & specify the Amazon EKS Amazon ECR image repository in the `image.repository: <Amazon EKS Amazon ECR image repository>` parameter. This needs to be done only for AWS Gov Cloud deployments.
```

Comment:
Uncomment and specify the Amazon EKS Amazon ECR image repository in the `image.repository: <Amazon EKS Amazon ECR image repository>` parameter. This is required for AWS GovCloud deployments

Reply:
Updated